logic/src/commonMain/kotlin/com/wire/kalium/logic/data/event/Event.kt

```diff
@@ -84,7 +86,12 @@ sealed class Event(open val id: String) {
     ) : Event(id) {
         data class FileSharingUpdated(
             override val id: String,
-            val status: String
+            val model: ConfigsStatusModel
+        ) : FeatureConfig(id)
+
+        data class MLSUpdated(
+            override val id: String,
+            val model: MLSModel
```

Comment:
isn't better to have it like MLSConfig? I think MLSModel is not explanatory enough.

Reply:
Every other feature config is called *Model in the logic module. I'd prefer rename this in separate refactoring PR.